lib/travis/logs/app.rb

```diff
@@ -56,6 +57,21 @@ def initialize(existence = nil, pusher = nil)
       get "/uptime" do
         status 204
       end
+
+      post "/logs/:id/clear" do
```

Comment:
My reasoning is that it just blanks out the log, the log record is still in the database. That might just be an implementation detail though?